README.md

```diff
@@ -246,7 +247,42 @@ Uncomment and edit the below line in the script to use Cmder config even when la
 # CMDER_ROOT=${USERPROFILE}/cmder  # This is not required if launched from Cmder.
 ```
 
-### Integrating Cmder with [Hyper](https://github.com/zeit/hyper), [Microsoft VS Code](https://code.visualstudio.com/), and your favorite IDEs 
+### Handling with custom arguments when using init.bat
```

Comment:
Since the actual flag comparison should be done in `/config/user-profile.cmd` and not `init.bat`, I suggest not mentioning `init.bat` as the arguments handler; instead advertise `user-profile.cmd` as where the arguments are handled.

Reply:
The original intention of this function is to handling different entries with different behaviors.
`/config/user-profile.cmd` is one of the entry. The other entries (eg. vsCode) need to pass arguments to init.bat to get the specific environment with specific behavior.

Graph: 
`/config/user-profile.cmd` with `argA` (=> init.bat (hidden for user I think)) => cmder environment (with Behavior A)
`vsCode internal terminal` with `argB`=> init.bat => **shell** environment (with Behavior B)
`other` with `argC` => init.bat => **shell** environment (with Behavior C)

Reply:
@xiazeyu I am utterly confused by your example.

Reply:
@daxgames Your understanding in https://github.com/cmderdev/cmder/pull/1758#discussion_r217885586 is correct. But it's just the current design.

@DRSDavidSoft suggests an further idea that args can be passed BOTH via init.bat and via cmder.exe(the launcher, he said he could have a try.), instead of only parse args passed to init.bat. 
Which means passing arguments to the launcher is also a way to start from different entries. (instead of only by init.bat)